apps/vaporgui/main.cpp

```diff
@@ -171,9 +174,17 @@ int           main(int argc, char **argv)
         setenv("PYTHONHOME", phome.c_str(), 1);
     }
     MyBase::SetDiagMsg("PYTHONHOME = %s", phome.c_str());
-
 #endif
 
+// Programatically set the hdf5 plugin path
+#ifndef WIN32
+	string plugins = Wasp::GetResourcePath("lib/hdf/HDF_Group/HDF5/1.12.2/lib/plugin");
+	H5PLreplace(plugins.c_str(), 0);
+#else
+    string envVar = "HDF5_PLUGIN_PATH=" + Wasp::GetResourcePath("plugin");
```

Comment:
The .so files should be configured so that they can be moved. If you look at the installer generator code, it ensures that our third party libraries are moveable.

Reply:
You're right, that worked.  The plugins are now in share/plugins.